tests/fast-integration/eventing-test/eventing-test-prep.js

```diff
@@ -110,21 +121,12 @@ describe('Eventing tests preparation', function() {
   it('Prepare JetStream data configmap', async function() {
     // Create a configmap that contains stream data for jetstream so that during the test,
     // we can verify that the stream was not affected/recreated
-    debug('expose the eventing-nats service with an apirule');
-    const vs = await createApiRuleForService(eventingNatsApiRuleAName,
-        kymaSystem,
-        eventingNatsSvcName,
-        8222);
-    const vsHost = vs.spec.hosts[0];
-
-    debug('Creating configmap with JetStream stream info');
-    const streamInfo = await getJetStreamStreamData(vsHost);
+    debug('Creating eventing test data configmap with JetStream stream info');
+    const streamInfo = await getJetStreamStreamData(natsApiRuleVSHost);
```

Comment:
Shall we also check if the stream name is correct before saving the data? like `streamName == "sap"`

Reply:
did you update anything here?

Reply:
yes, please take a look at inside of the function `getJetStreamStreamData`. I embedded check inside the function.